### PR TITLE
Fetches nuclear production for JP-KN

### PR DESCRIPTION
**Pull request by Kongkille**
Based on research from #541 (thanks a lot for the help)

It fetches from https://www.kepco.co.jp/energy_supply/energy/nuclear_power/info/monitor/live_unten/, which seems to update every 20 minutes or so. The scraper will most certainly fail if changes are made to the HTML, it's not robust at all.

It tries to match the nuclear production with solar production if it does not find a match within 30 minutes, it will fail.

Let me know if there are any code or methodology improvements I should implement! :)

### CODE REVIEW

File: config/zones.json [hunk 1]
```diff
@@ -2937,7 +2937,7 @@
     ],
```
Comment (FelixDQ) [inline config/zones.json:2937]: Remember to add you name here 😄

File: parsers/JP_KN.py [hunk 2]
```diff
@@ -0,0 +1,151 @@
...
+    JP_data = JP_fetch_production(zone_key, session, target_datetime, logger)
+    nuclear_mw, nuclear_datetime = get_nuclear_production()
+    latest = JP_data[-1] #latest solar data is the most likely to fit with nuclear production
+    diff = None
+    if nuclear_datetime > latest["datetime"]:
+        diff = nuclear_datetime-latest["datetime"]
+    else:
+        diff = latest["datetime"]-nuclear_datetime
+    if abs(diff.seconds) > 30 * 60:
+        raise Exception("Difference between nuclear datetime and solar data is too large")
```
Comment (FelixDQ) [inline parsers/JP_KN.py:46]: A nitpick, but it's the difference between nuclear data and the rest of the data right (solar + unknown)? Maybe change the comments to be "JP data" or "production data".

File: parsers/JP_KN.py [hunk 3]
```diff
@@ -0,0 +1,151 @@
...
+
+    latest["production"]["nuclear"] = nuclear_mw
+    latest["production"]["unknown"] = latest["production"]["unknown"] - nuclear_mw
+    return latest
+
+# https://stackoverflow.com/questions/50655738/how-do-i-resolve-a-tesseractnotfounderror
```
Comment (FelixDQ) [inline parsers/JP_KN.py:52]: Was this link meant to be here? Maybe move it up to the import

**Comment by Kongkille** [comment]
@FelixDQ 

For installing tesseract (I believe it is already installed, but here they are for linux)
- sudo apt update
- sudo apt install tesseract-ocr
- sudo apt install libtesseract-dev

And for installing the japanese language (I am not sure this works, since I tested on Mac)

- sudo apt-get install tesseract-ocr-jpn

**Pull request merged**